packages/connect/e2e/__fixtures__/getAddressMultisig.ts

```diff
@@ -1,20 +1,40 @@
-const multisig = {
+import { MultisigPubkeysOrder } from "@trezor/protobuf/src/messages";
```

Comment:
I would expect the tests here to pass as they are now but even providing `pubkeys_order: MultisigPubkeysOrder.LEXICOGRAPHIC` they produce different address.

I can replicate it using trezorctl and it works there, but not able to make it work with connect yet.

Here instructions how to replicate it with trezorctl https://github.com/trezor/trezor-suite/pull/15865/files#diff-ecf90d5838aea6392821666a8ac3341def968255c5d9e6043f4f5f2307c815fe